tests/test_validation.py

```diff
@@ -1037,7 +1037,10 @@ def step_3(self):
                             pass
                         else:
                             keyword = keyword_all_clear[j]
-                            self.assertEqual(row[j], test_dict[keyword][0], msg = 'This is the keyword ' + keyword)
+                            try:
```

Comment:
What is the purpose of this try/except block?  Isn't the expected result completely determined?